client/rpc/debug/src/lib.rs

```diff
@@ -70,16 +68,22 @@ where
 	C::Api: EthereumRuntimeRPCApi<B>,
 {
 	// Asumes there is only one mapped canonical block in the AuxStore, otherwise something is wrong
-	fn load_hash(&self, hash: H256) -> RpcResult<Option<BlockId<B>>> {
-		let hashes = match fc_consensus::load_block_hash::<B, _>(self.client.as_ref(), hash)
+	pub fn load_hash(client: &C, hash: H256) -> RpcResult<Option<BlockId<B>>> {
```

Comment:
I added `load_hash`, `is_canon` and `load_transactions` as part of the Debug module from the get-go but ideally they should be part of an independent module in Frontier - in fact they are around already in https://github.com/paritytech/frontier/blob/master/client/rpc/src/eth.rs#L318.

Then we should be able to remove them completely from Moonbeam and add them as a dependency.

Reply:
Submitted https://github.com/paritytech/frontier/pull/331, once there we should be able to remove those locally.